tests/test-admin-dashboard.php

```diff
@@ -58,6 +58,40 @@ public function test_lowly_user() {
 		$this->assertTrue( isset( $wp_meta_boxes['dashboard-user']['normal']['high']['pb_dashboard_widget_book_permissions'] ) );
 	}
 
+	/**
+	 * @group dashboard
+	 */
+	public function test_lowly_user_remove_healthy_and_wp_news_widgets() {
+		global $wp_meta_boxes;
+		// Mock dashboard user, by default the dashboard will be admin in this test
+		$wp_meta_boxes['dashboard-user'] = [
```

Comment:
Tests actually runs on Admin's Dashboard instead User's Dashboard. I mocked the user's dashboard.
TODO in the future: create tests for other users roles.